src/modules/dimension.ts

```diff
@@ -270,3 +272,87 @@ export const isTimeDimensionId = (
     dimensionId: DimensionRecord['dimension']
 ): dimensionId is TimeDimensionId =>
     (TIME_DIMENSION_IDS as readonly string[]).includes(dimensionId)
+
+type FormatDimensionIdParams = {
+    dimensionId: DimensionId
+    programStageId?: string
+    programId?: string
+    outputType?: SavedVisualization['outputType']
+}
+export const formatDimensionId = ({
```

Comment:
This was already ported as `getFullDimensionId` and as far as I can tell it's doing exactly the same but with `outputType` renamed to `inputType`.